src/Whoops/Util/TemplateHelper.php

```diff
@@ -106,9 +111,8 @@ public function breakOnDelimiter($delimiter, $s)
      */
     public function shorten($path)
     {
-        $dirname = dirname(dirname(dirname(dirname(dirname(dirname(__DIR__))))));
-        if ($dirname != "/") {
-            $path = str_replace($dirname, '&hellip;', $path);
+        if ($this->applicationRootPath != "/") {
```

Comment:
I wouldnt define a default of '/' here .. take the root path in account when defined and give it an empty/null default. The prettypagehandler should be in charge to provide one